lib/handlers/handler.ts

```diff
@@ -220,6 +224,8 @@ export abstract class APIGLambdaHandler<CInj, RInj extends BaseRInj, ReqBody, Re
     )
   }
 
+  protected abstract afterHandler(metric: MetricsLogger, response: Res, requestStart: number): void
```

Comment:
let's not make this abstract, not every handler will need to implement it. (lol we only have 1 handler here)

Reply:
sg, will do